client/ayon_sitesync/version.py

```diff
@@ -1,3 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Why was removed?

Reply:
It is added dynamically to the package only, I don't like to have 2 places to bump up version number.

Reply:
Please keep it for running from code.

Reply:
Ok, I put it back.